prow/plugins/jira-linker/jira-linker.go

```diff
@@ -87,7 +87,17 @@ func handle(gc githubClient, log *logrus.Entry, config plugins.JiraLinker, event
 		}
 
 		if !hasLabel {
-			gc.CreateComment(org, repo, event.Number, commentForTicket(jiraLink(config.JiraBaseUrl, ticketName)))
+			jiraServerURL := config.JiraBaseUrl
+			out:
```

Comment:
Nice fix!

One minor issue to fix though: when we break out from the loop, that means that we've found our match, and we don't need to search anymore. Thus, the `out` label should probably go _after_ the loop, instead of _before_ it (eg move `out:` from `L91` to `L99`).

As written, I believe that this would end up with any case where we find a match going into an infinite loop, since it will:

1. L92: Start scanning for overrides,
2. L95: Find a match,
3. L96: `break out` and return back to `L91`,
4. L92: Start scanning for overrides again,

etc etc forever.

Reply:
I'm not sure if it goes before or after anymore. This blog (and others) mention that the label goes before the loop. https://medium.com/golangspec/labels-in-go-4ffd81932339